### DISCUSSION TITLE
(Unsure) Disk burning

### DISCUSSION THREAD

**Discussion by anonimo82** [doublecmd/doublecmd#1828] [Suggestions]
Hello everyone.
As in subject, I'm unsire if this is in scope with DoubleCMD, but I'd like to suggest disk burning (or at least ISO creation, then physically burning with somethingn else).
I'd like to burn Audio CDs, Video DVDs/BDs, and Data disks.
I'm not interested in authoring such media (such as menus etc) but it could be an idea.

Please, tell me if this is out of scope.

Thanks in advance for any help.
Regards, have a nice day.

**Comment by j2969719** [comment]
in the golden days of winxp/nero no joke there was such a wfx plugin for totalcmd on one of my shovelware cd

**Comment by Skif-off** [comment]
This is not a joke, sounds like [Neropanel](https://totalcmd.net/plugring/neropanel.html).

IMHO, console utilities or GUI with command line parameters are more practical and flexible...

**Reply by j2969719** [reply]
>Neropanel

kek

btw iirc gvfs can also burn disks but i dont have a live burner anymore to check

**Comment by alexx2000** [comment]
Also [TCBurner](https://totalcmd.net/plugring/tcburner.html). But the epoch of optical discs is over.

**Reply by anonimo82** [reply]
TCBurner looks good, is it there any way to use it with Linux?
I mean, with wine. Should I run the whole DoubleCMD with Wine, or can I just use the plugin with it?

Thanks for any help.
Regards, have a nice day.

**Reply by Skif-off** [reply]
> is it there any way to use it with Linux?

Only Windows version with Wine (but without guarantees in any case).
